RadarSDK/RadarReplay.h

```diff
@@ -7,7 +7,7 @@
 
 #import "Radar.h"
 
-@interface RadarReplay : NSObject <NSCoding>
+@interface RadarReplay : NSObject <NSCoding, NSSecureCoding>
```

Comment:
NSSecureCoding inherit from NSCoding, so you don't need both

Reply:
oh yeah lol